### ISSUE TITLE
example not working

### ISSUE THREAD

**Issue by xtlc** [adamchainz/django-htmx#27]
I was curios to try this package as I wanted to dig into HTMX, but I could not get the example project up and running. A little showstopper in the requirements:

- django_htmx is missing in the projects requirements.txt

But now I am stuck with these errors:

> File "..\venv\lib\site-packages\django\utils\module_loading.py", line 20, in import_string
>     return getattr(module, class_name)
> AttributeError: module 'django_htmx' has no attribute 'HtmxMiddleware'
> 
> File "..\venv\lib\site-packages\django\utils\module_loading.py", line 22, in import_string
>     raise ImportError('Module "%s" does not define a "%s" attribute/class' % (
> ImportError: Module "django_htmx" does not define a "HtmxMiddleware" attribute/class

Also, is there a particular reason this would not work with Django 3.1 but only 3.0.12?

**Comment by danjac** [comment]
It looks like the version available in PyPi (0.14) bears no resemblance to the documentation. I'm seeing the following files installed:

__init__.py  __pycache__  static  templates  templatetags  views.py

**Comment by xtlc** [comment]
I guess this closed issue is the explenation: [Link](https://github.com/adamchainz/django-htmx/issues/9#issuecomment-747973558).
Pypi will be updated once a PR is merged into the package.

**Comment by adamchainz** [comment]
Indeed this repo is not what's on PyPI yet. To use the example follow its README which installs the local version of the repo not from PyPI, with the `-e ..` part of `pip install`: https://github.com/adamchainz/django-htmx/tree/main/example
